packages/wrangler/src/sites.tsx

```diff
@@ -13,23 +14,21 @@ import type { KeyValue } from "./kv";
 import type { XXHashAPI } from "xxhash-wasm";
 
 /** Paths to always ignore. */
-const ALWAYS_IGNORE = ["node_modules"];
-const HIDDEN_FILES_TO_INCLUDE = [
+const ALWAYS_IGNORE = new Set(["node_modules"]);
+const HIDDEN_FILES_TO_INCLUDE = new Set([
   ".well-known", // See https://datatracker.ietf.org/doc/html/rfc8615
-];
+]);
+const FIVE_MINUTES = 1000 * 60 * 5;
```

Comment:
Should this actually just be 60 * 5 instead of 1000 * 60 * 5? [the docs](https://developers.cloudflare.com/workers/runtime-apis/kv/#creating-expiring-keys) say it's in seconds, but I haven't tested it myself so I'm not sure

Reply:
Good spot @caass. So you are right that the API does say seconds since the epoch.
But `Date.now()` returns milliseconds since the epoch, so `Date.now() + FIVE_MINUTES` is the correct number of millisecs.

So if the docs are correct, then it is actually the bit where we assign the FIVE_MINUTE_LATER to the `expiration` field that needs fixing.

Reply:
oof good catch(es), that would have been a bad mistake. thank you!